Armor/armor.js

```diff
@@ -27,6 +27,7 @@
 // Modify param namespace to make room for armor
 ParamType.ARM = 11
 ParamType.COUNT = 12
+var armorDamageDiv = 2;
```

Comment:
Stray line?  Looks like it was probably the original config var for `fractionDamage` and is now unused.  Can ignore for now because no side effects.

Reply:
ah yea mustve missed that one. used it when prototyping, forgot it when i was cleaning up